torntools/scripts/content/loader/ttCasinoStatistics.js

```diff
@@ -1,5 +1,9 @@
 requireDatabase().then(() => {
 	console.log("TT - Casino Statistics");
+	if (!window.location.toString().includes("Lottery")) {
+		addNetTotal("overall");
+		addNetTotal("your");
+	};
 	window.addEventListener("hashchange", () => {
```

Comment:
Since the hash listener is only needed for the bookies page, can it be done so it's only added there?

Reply:
Done.